src/components/app/app.html

```diff
@@ -1 +1,4 @@
-<div class="app">hi, <%= name %>!</div>
+<div class="app">
+  <div class="app__drawer-point"></div>
+  <div class="app__toolbar-point toolbar"></div>
```

Comment:
toolbar component should not be here -- it should mount with toolbar js component